crates/bevy_asset/src/io/file/sync_file_asset.rs

```diff
@@ -160,10 +160,7 @@ impl AssetReader for FileAssetReader {
         }
     }
 
-    async fn is_directory<'a>(
-        &'a self,
-        path: &'a Path,
-    ) -> std::result::Result<bool, AssetReaderError> {
+    async fn is_directory<'a>(&'a self, path: &'a Path) -> Result<bool, AssetReaderError> {
```

Comment:
Sorry, this isn't strictly in scope of this PR, but was also a warning when I ran `cargo test -p bevy_gizmos`, I'm happy to remove changes to this file if it's controversial.

Reply:
This is trivial; I don't mind it getting cleaned up here.